### PR TITLE
simplify readme

### PR DESCRIPTION
**Pull request by sharkAndshark**
- [x] add slack chat icon 
- [x] merging maplibre/leaflet/mapbox demos into `using with clients`
- [x] cut off a lot contents 
- [x] downgrade contents from h1 to h2 
- [x] remove TOC .[GitHub will now automatically generate a table of contents in the header when there are 2 or more headings](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/) 
- [x] add link to the book in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
...
 
-# Usage
+## Usage
 
-Martin requires at least one PostgreSQL [connection string](#postgresql-connection-string) or a [tile source file](#mbtile-and-pmtile-sources) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
+### PostGIS sources
+Martin requires at least one PostgreSQL [connection string](https://maplibre.org/martin/PostgreSQL-Connection-String.html) or a [tile source file](https://maplibre.org/martin/MBTile-and-PMTile-Sources.html) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
 
 ```shell
 martin postgresql://postgres@localhost/db
```
Comment (nyurik) [inline README.md]: should this also use `user@host` style of a link, same as above?

File: README.md [hunk 2]
```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
...
+| `GET`  | `/`                                    | Status text, that will eventually show web UI                                                             |
+| `GET`  | `/catalog`                             | [List of all sources](https://maplibre.org/martin/source-list.html)                                       |
+| `GET`  | `/{sourceID}`                          | [Source TileJSON](https://maplibre.org/martin/table-sources.html#table-source-tilejson)                   |
+| `GET`  | `/{sourceID}/{z}/{x}/{y}`              | [Source Tiles](https://maplibre.org/martin/table-sources.html#table-source-tiles)                         |
+| `GET`  | `/{sourceID1},...,{nameN}`             | [Composite Source TileJSON](https://maplibre.org/martin/composite-sources.html#composite-source-tilejson) |
+| `GET`  | `/{sourceID1},...,{nameN}/{z}/{x}/{y}` | [Composite Source Tiles](https://maplibre.org/martin/composite-sources.html#composite-source-tiles)       |
+| `GET`  | `/health`                              | Martin server health check: returns 200 `OK`                                                              |
+
+## Using with clients
```
Comment (nyurik) [inline README.md]: I think all the usage with various clients should be deleted too - lets keep it only in the book.

File: README.md [hunk 3]
```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
...
+Martin requires at least one PostgreSQL [connection string](https://maplibre.org/martin/PostgreSQL-Connection-String.html) or a [tile source file](https://maplibre.org/martin/MBTile-and-PMTile-Sources.html) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
 
 ```shell
 martin postgresql://postgres@localhost/db
 ```
 
 Martin provides [TileJSON](https://github.com/mapbox/tilejson-spec) endpoint for each [geospatial-enabled](https://postgis.net/docs/using_postgis_dbmanagement.html#geometry_columns) table in your database.
 
-# API
+### MBTiles and PMTiles sources
```
Comment (nyurik) [inline README.md:76]: I think at the end of each section we might provide a link to the relevant section of the book?  Or as part of the text?

File: README.md [hunk 4]
```diff
@@ -2,55 +2,16 @@
 
 [![CI](https://github.com/maplibre/martin/workflows/CI/badge.svg)](https://github.com/maplibre/martin/actions)
 ![Security audit](https://github.com/maplibre/martin/workflows/Security%20audit/badge.svg)
-
+[![Slack chat](https://img.shields.io/badge/Chat-on%20Slack-brightgreen)](https://slack.openstreetmap.us/)
```
Comment (nyurik) [inline README.md]: thanks!  Want to make it first, or else we have two off-green items next to each other.

File: README.md [hunk 5]
```diff
@@ -2,55 +2,16 @@
...
-  * [Other useful commands](#other-useful-commands)
-* [Recipes](#recipes)
-  * [Using with DigitalOcean PostgreSQL](#using-with-digitalocean-postgresql)
-  * [Using with Heroku PostgreSQL](#using-with-heroku-postgresql)
-<!-- TOC -->
-
-# Requirements
+## Requirements
 
 Martin requires PostGIS 3.0+.  PostGIS 3.1+ is recommended.
```
Comment (nyurik) [inline README.md]: this should probably be corrected -- PostGIS is only needed when using Martin with PostgreSQL, not for mbtiles/pmtiles

File: README.md (hunk 1 above)
Comment (sharkAndshark) [reply README.md]: The styles are same now

File: README.md (hunk 2 above)
Comment (sharkAndshark) [reply README.md]: Removed

File: README.md (hunk 4 above)
Comment (sharkAndshark) [reply README.md]: Make it first now

File: README.md (hunk 5 above)
Comment (sharkAndshark) [reply README.md]: Description updated

**Review by nyurik — APPROVED** [review]

**Comment by nyurik** [comment]
Absolutely great, thank you! I made a few finishing touches, nothing major. In a subsequent PR, we may want to add many links to the various book pages from multiple readme sections, e.g. to the API section, to the PG, to MBTiles, etc.  But no biggie

**Pull request merged**